skimage/filters/tests/test_lpi_filter.py

```diff
@@ -10,11 +10,7 @@
 from skimage._shared._warnings import expected_warnings
 
 
-if (Version(np.__version__) >= '1.15.0' and
-        Version(scipy.__version__) <= '1.1.0'):
-    SCIPY_ND_INDEXING_WARNING = 'non-tuple sequence for multidimensional'
-else:
-    SCIPY_ND_INDEXING_WARNING = None
+SCIPY_ND_INDEXING_WARNING = None
```

Comment:
Want to just remove this all together?

Reply:
Yes, it is ready, just wanted to see the CI full green before pushing :stuck_out_tongue_closed_eyes:

Reply:
OK, can't wait :smile: